pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/DependencyOrder.java

```diff
@@ -35,6 +27,14 @@
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.Logging;
 
+import java.io.File;
```

Comment:
This file has no other change except this. Did we even need to change it then?
What if org.gradle.api.... has a Collection class? It should be able to override the java.util.Collection. But it cannot because of this import order.

I'm not saying it's a good idea. I'm just saying the original order seems correct to me.

Reply:
Default IntelliJ does this formatting. When I was fixing imports it happened.